cmd/ui.go

```diff
@@ -114,18 +114,23 @@ func printExecutionDescription(
 		for _, out := range outputs {
 			desc := out.Description()
 			if desc == engine.IngesterDescription {
-				if len(outputs) != 1 {
-					continue
-				}
-				desc = "-"
+				continue
+			}
+			if ok, v := checkWebDashboardDescription(desc); ok {
+				fmt.Fprintf(buf, "web dashboard: %s\n", valueColor.Sprint(v))
```

Comment:
It does print the following string on my side.
```
web dashboard: (:5665) http://127.0.0.1:5665
```
Do we need to repeat the URL twice?

On my side, it also prints the line without any padding from the left border of the terminal which doesn't seem optimal.

Reply:
yes, this appears for me too. In xk6-dashboard v0.7.0, there will no longer be redundant information, only the URL.
Otherwise, I increased the indentation by 2 characters (the "web dashboard" string is so much longer than the previous field size). Please let me know if I should increase the indentation by one more character for the other fields as well?
​

Reply:
![2023-12-18-112803](https://github.com/grafana/k6/assets/2103732/4327d479-5d79-42cb-9ec3-864ebe4ffc28)

The blue line is where the terminal's tile finishes.

Reply:
yes. I can indent each lines with one more space, but I'm worry about indent became too deep and will cause line breaks on longer lines. So, should I add one more space?

Reply:
or can we reconsider using "web dashboard" name? I'd prefer simply "dashboard" (yes it is general, but probably we will have not other dashboard in near future)

Reply:
>but I'm worry about indent became too deep and will cause line breaks on longer lines

The metrics summary is already longer so I don't think it should be a problem having them a bit more indented. Do you have any specific line you're thinking it could break?

>or can we reconsider using "web dashboard" name? 

As we are part of a company that builds tools for graphing data, everything is a dashboard, so I prefer `web dashboard` as it is a bit clearer. We already have dashboards for other outputs, and they are all Grafana-based so during communications, verbal or written, having `web` prefix should clarify the context. In any case, if you want to re-discuss it open the question to the working group.

Reply:
>  Do you have any specific line you're thinking it could break?
No
I added one more space, so "web dashboard" no indented by one space from left.